app/models/given_answer.rb

```diff
@@ -14,4 +14,8 @@
 class GivenAnswer < ApplicationRecord
   belongs_to :question
   validates :question_id, :user_id, :answers, presence: true
+
+  def user_name
+    User.find(user_id).full_name
```

Comment:
This shouldn't be here, because the GivenAnswer shouldn't be aware of the structure of the User model.

ie: if the User model changes, you'll also break the GivenAnswer model, which wouldn't be expected.

This method is good, but I think it should go to a presenter. ;)